tests/Unit/NumberToTextTest.php

```diff
@@ -1,43 +0,0 @@
-<?php
```

Comment:
this file cause some issues for folks, which has been fixed in most recent codebase. i'll plan to add this script back after this code gets into codebase.

Reply:
Ok, I am not sure how it got tangled up in my commit.